recipes/impactutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7" %}
+
+package:
+  name: impactutils
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/earthquake-impact-utils/archive/{{ version }}.tar.gz
+  sha256: 896081c5e7528c0d5ab11415dcd52037bcc0f5af0756956fa3ef74f68a2b34e1
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python >=3.4
+    - beautifulsoup4 >=4.4.0
+    - cartopy >=0.14.0
+    - fiona >=1.6.0
+    - h5py >=2.2
+    - matplotlib <2.0
+    - numpy >=1.0
+    - jupyter
+    - pandas >=0.8.1
+    - paramiko >=1.15.2
+    - pycrypto >=2.6.1
+    - seaborn >=0.6.0
+    - shapely >=1.5.11
+
+test:
+
+  imports:
+    - impactutils
+
+about:
+  home: https://github.com/usgs/earthquake-impact-utils
+  license: Public Domain
```

Comment:
Add the `license_file: LICENSE.md`

Reply:
I have this file:
https://github.com/usgs/earthquake-impact-utils/blob/master/LICENSE.md

Do you want me to add that to the conda-forge repo?